index.php

```diff
@@ -1,18 +1,18 @@
 <!DOCTYPE HTML>
-<!-- Author: Ashley Burnett -->
 
 <?php
 //Used for browser title and for META tags.
+include 'functions.php';
 $title = "Medieval Latin Lessons - The National Archives"
 ?>
 
 <!-- no-js class is needed for Modernizr -->
 <html lang="en-gb" class="no-js">
 <head>
     <title><?= $title ?></title>
```

Comment:
I'm afraid we do not use the php shorthand way. Please update to <?php ?> rather than <?=  ?>